### ISSUE TITLE
Many warnings from `peak_classification_bayes` 

### ISSUE THREAD

**Issue by JoranAngevaare** [XENONnT/straxen#974]
**Describe the bug**
Getting loads of warnings like this:
```
/opt/XENONnT/anaconda/envs/XENONnT_2022.03.5/lib/python3.8/site-packages/numpy/core/fromnumeric.py:86: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
```
probably can be fixed easily by using a structured array instead of a dictionary in `peak_classification_bayes`

**To Reproduce**
Insert the MWE of how to reproduce the error
```python
st=cutax.xenonnt_v7()
assert not st.is_stored('031043', 'peak_classification_bayes')
st.make('031043', 'event_w_bayes_class')
```

**Versions**
Please add the output of:
```
 module version                                                                             path  git
 python  3.8.12                          /opt/XENONnT/anaconda/envs/XENONnT_2022.03.5/bin/python None
straxen   1.6.0 /opt/XENONnT/anaconda/envs/XENONnT_2022.03.5/lib/python3.8/site-packages/straxen None
  strax   1.2.0   /opt/XENONnT/anaconda/envs/XENONnT_2022.03.5/lib/python3.8/site-packages/strax None
  cutax   1.6.0                                /dali/lgrandi/xenonnt/software/cutax/latest/cutax None
```

**Comment by ahiguera-mx** [comment]
Hi @JoranAngevaare 
Do you mean return an structured array like this
```
result['time'] = peaks['time']
result['endtime'] = peaks['time'] + peaks['dt'] * peaks['length']
result['ln_prob_s1'] = ln_prob_s1
result['ln_prob_s2'] = ln_prob_s2

return result
```

That does not seem to remove the warning

**Comment by ahiguera-mx** [comment]
[This line](https://github.com/XENONnT/straxen/blob/ffcf06ad86471caf11cc831f2ff68d70b59464af/straxen/plugins/bayes_peak_classification.py#L137) is causing the warning, a potential solution can be  

```
_lnposterior_sumsamples = np.array([lnposterior_sumsamples, np.log(class_prior)[np.newaxis, ...]], dtype=object)
lnposterior_sumsamples = np.sum(_lnposterior_sumsamples, axis=0)
```
but because some chunks have only one peak as mentioned [here](https://github.com/XENONnT/straxen/issues/977)  this solution throws an error dtype=object is not support. So at the moment given the feature of one peak chunk we will have this warning still

**Comment by ahiguera-mx** [comment]
@JoranAngevaare 
I already tried that and that is not causing the warning, the warning is about [this](https://github.com/XENONnT/straxen/issues/974#issuecomment-1074405620)

**Comment by JoranAngevaare** [comment]
fixed in #1015
